packages/rolldown/build.ts

```diff
@@ -1,34 +1,41 @@
+import colors from 'ansis';
 import { globSync } from 'glob';
 import fs from 'node:fs';
 import nodePath from 'node:path';
 import pkgJson from './package.json' with { type: 'json' };
-import { colors } from './src/cli/colors';
-import {
-  defineConfig,
-  OutputOptions,
-  type Plugin,
-  rolldown,
-} from './src/index';
-
-const IS_RELEASING_CI = !!process.env.RELEASING;
-const IS_BUILD_WASI_PKG = !!process.env.WASI_PKG;
-
-const outputDir = IS_BUILD_WASI_PKG
-  ? nodePath.resolve(__dirname, '../wasi/dist')
+import { build, BuildOptions, defineConfig, type Plugin } from './src/index';
+
+const isCI = !!process.env.CI;
+const isReleasingCI = !!process.env.RELEASING;
+const isBrowserBuild = !!process.env.BROWSER_PKG;
+
+const outputDir = isBrowserBuild
+  ? nodePath.resolve(__dirname, '../browser/dist')
   : nodePath.resolve(__dirname, 'dist');
 
+const bindingFile = nodePath.resolve('src/binding.js');
+const bindingFileWasiBrowser = nodePath.resolve(
+  'src/rolldown-binding.wasi-browser.js',
+);
+
 const shared = defineConfig({
   input: {
     index: './src/index',
-    cli: './src/cli/index',
-    'parallel-plugin': './src/parallel-plugin',
-    'parallel-plugin-worker': './src/parallel-plugin-worker',
-    'experimental-index': './src/experimental-index',
-    'parse-ast-index': './src/parse-ast-index',
+    ...isBrowserBuild ? {} : {
```

Comment:
If browser build doesn't requiest the exactly same structure as `rolldown` package, you might add new a `build.mts` file in `browser/src` to do the browser build only.